src/app/(sidebar)/endpoints/[[...pages]]/page.tsx

```diff
@@ -262,7 +266,7 @@ export default function Endpoints() {
     isValidReqFields = missingReqFields.length === 0;
 
     // Checking if there are any errors
-    isValid = formError.tx?.result === "success" || isEmptyObject(formError);
```

Comment:
we don't need to check for this one since I added [this conditional](https://github.com/stellar/laboratory/pull/911/files#r1690766077) to prevent it from being added